apps/api/src/index.ts

```diff
@@ -106,14 +106,14 @@ async function initAPI() {
     // #endregion
 
     // #region Start Server
-    server.listen(settings.port, () => {
-        console.log(`Server listening on port ${settings.port}`);
+    server.listen(settings?.port, () => {
+        console.log(`Server listening on port ${settings?.port}`);
         openInBrowser();
     });
     // #endregion
 
     // #region Start Version Check
-    StartVersionCheck();
+    await StartVersionCheck();
```

Comment:
`StartVersionCheck` shouldn't need to be awaited here since it's async properties are only used internally in the function, since it runs on a loop.

Reply:
its being awaited because of the GET request happening in it